charts/sfapm-python3/values.yaml

```diff
@@ -179,6 +179,9 @@ vizbuilder_celery:
   signature:
     replicaCount: 1
     command: "celery -A vizbuilder worker -l $LOG_LEVEL -Q signature -c10 $CELERY_OPTS"
+  provisiontemplate:
+    replicaCount: 1
+    command: "celery -A vizbuilder worker -l $LOG_LEVEL -Q provisiontemplate -c10 $CELERY_OPTS"
```

Comment:
change -c 10 to -c 1, it should run only 1 thread if you want only one task should be executed at a time